docs/howtos/another_kolibri_instance.md

```diff
@@ -0,0 +1,55 @@
+## Running Another Kolibri Instance Alongside the Development Server

```

Comment:
Even though we do not have a formal style guide for developer docs, we don't use the the title case for the headings, but the sentence case (just the first word in the sentence is capitalized).